python/rmm/rmm/__init__.py

```diff
@@ -52,3 +52,13 @@
     "should_log",
     "unregister_reinitialize_hook",
 ]
+
+
+def __getattr__(name):
+    if name == "_lib":
+        import importlib
+
+        module = importlib.import_module("rmm.pylibrmm")
+        return module
+    else:
```

Comment:
So is the plan to add a deprecation warning here after we merge the PRs to other RAPIDS repos to use pylibrmm?

Reply:
Yes